jni/lib/src/jni.dart

```diff
@@ -256,3 +255,54 @@ extension ProtectedJniExtensions on Jni {
     return lookup;
   }
 }
+
+extension AdditionalEnvMethods on Pointer<GlobalJniEnv> {
```

Comment:
(The GitHub interface is not good at doing showing diffs, please only move around things if really needed.)